src/components/Pages/Members/Members.module.css

```diff
@@ -0,0 +1,3 @@
+.wrapper {
+    width: 800px;
```

Comment:
I see it twice, use css variable for content width

Reply:
Fixed. 
```suggestion
        width: var(--width-content);
```